circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/Compression.scala

```diff
@@ -53,6 +53,20 @@ object Compression {
       bytes
     }
 
+  /*
+  Since we are encoding JSON for Ser <: AnyRef types - they will start with '{' char for Json objects or with '[' char for Arrays.
```

Comment:
Slight nuance here... what about `String`s? they extend `AnyRef` as well
Ofc strings in Json start with `"` character, so not a problem here anyway

Reply:
yes, my mistake - let me add String to this description - " character is represented by 34 decimal so this one is also safe